js/events/contextmenu.js

```diff
@@ -36,6 +36,9 @@ const ContextMenu = Class.inherit({
     },
 
     _contextMenuHandler: function(e) {
+        if(touch || devices.isSimulator()) {
+            e.preventDefault();
```

Comment:
I am not sure that we should prevent each 'contextMenu' event on entire element that was targeted by our ContextMenu event because our event can be cancelled ("e.cancel = true;")

Reply:
I checked this code "e.cancel = true" instead of "e.preventDefault ()" but it didn't canceled the browser menu appears.